src/main/webapp/app/shared/metis/posting-reactions-bar/posting-reactions-bar.component.ts

```diff
@@ -11,6 +11,10 @@ const PIN_EMOJI_ID = 'pushpin';
 const PIN_EMOJI_UNICODE = '1F4CC';
 const ARCHIVE_EMOJI_ID = 'open_file_folder';
 const ARCHIVE_EMOJI_UNICODE = '1F4C2';
+const SPEECH_BALOON_ID = 'speech_balloon';
+const SPEECH_BALOON_UNICODE = '1F4AC';
```

Comment:
```suggestion
const SPEECH_BALLOON_UNICODE = '1F4AC';
```

Reply:
Nice catch, thanks!

Reply:
Did you forget to push this change?
You requested a re-review, but this is still not fixed.

Reply:
I'm sorry, seems like the process failed due to git hooks. It should be there now.

Reply:
GitHub still does not show any new commits.